### PR TITLE
v0.6.01

### PR DESCRIPTION
**Pull request by hmerrettICHK**
Adds a notification option for notifying when a new unit is created.

### CODE REVIEW

File: Flexible Learning/units_manage_addProcess.php [hunk 1]
```diff
@@ -75,20 +84,45 @@
         $file = $_FILES['file'] ?? null;
 
         // Upload the file, return the /uploads relative path
+<<<<<<< Updated upstream
         $data['logo'] = $fileUploader->uploadFromPost($file, $name);
+=======
+        $data['logo'] = $fileUploader->uploadFromPost($file, $name, $data['name']);
```
Comment (rossdotparker) [inline Flexible Learning/units_manage_addProcess.php]: Having fixed this here, are you possibly seeing the same error in editProcess? If so, worth fixing there.

Comment (rossdotparker) [reply Flexible Learning/units_manage_addProcess.php]: Also, I would test this change by creating a new unit, and making sure that the name of the image (as it appears in the database row for the unit) contains the name of the unit.

File: Flexible Learning/units_manage_addProcess.php [hunk 2]
```diff
@@ -75,20 +84,45 @@
...
+
+>>>>>>> Stashed changes
     // Create the record
     if (!$flexibleLearningUnitID = $unitGateway->insert($data)) {
         $URL .= '&return=error2';
         header("Location: {$URL}");
         exit;
     }
 
+<<<<<<< Updated upstream
```
Comment (rossdotparker) [inline Flexible Learning/units_manage_addProcess.php]: You've got quite a few merge issues, relating to upstream and stash. I think @SKuipers will be able to advise you on dealing with these.

**Comment by rossdotparker** [comment]
A neat and useful change, and looking pretty good. I've dropped a couple of comments. Once Sandra has reviewed, you've solved any issues, and Sandra has merged...please can you ask her to do the release and push to production at ICHK for final tweaks and tests? Thanks!

**Comment by SKuipers** [comment]
Hi Ross, I don't see the comments, be sure to click to submit your review if you started one.

Harry, looks like we may have missed some of the changes when merging your branch. You can see these in the areas separated by >>>>>> and <<<<<<. Check these out and see if you can resolve them, then commit and push your changes to this branch.

Thanks!

**Review by rossdotparker — CHANGES_REQUESTED** [review]

**Comment by hmerrettICHK** [comment]
New commit made, fixed upstream/stash errors and updated editProcess

**Review by SKuipers — APPROVED** [review]
Thanks Harry, this is looking good, I'm going to merge it and test it out now 👍

**Comment by SKuipers** [comment]
Pushed to production and updated, looking great! I've added you to receive all these notifications 😄

**Pull request merged**